src/js/post-911-gib-status/utils/helpers.jsx

```diff
@@ -78,6 +78,27 @@ export function benefitEndDateExplanation(condition, delimitingDate) {
   }
 }
 
+export function notQualifiedWarning() {
+  return (
+    <div className="usa-alert usa-alert-warning usa-content not-qualified">
+      <div className="usa-alert-body">
+        <h2>Currently Not Qualified</h2>
+        Our records show you don't qualify for the Post-9/11 GI Bill because:
+        <ul>
+          <li>You haven't served on active duty for at least 90 days after September 10, 2001, and/or</li>
+          <li>Your service component let us know that you didn't receive an honorable discharge</li>
+          <li>If the information in our records is not accurate, please call us at 888-442-4551 (888-GI-BILL-1) from 8 a.m. to 7 p.m (ET).</li>
```

Comment:
@rroueche Wondering if this last bullet point should actually be a line on its own since it's not one of the reasons the user may not qualify because of.